packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -277,6 +278,10 @@ export class L1TransportServer extends BaseService<L1TransportServerOptions> {
             throw new Error(`Unknown transaction backend ${backend}`)
         }
 
+        // Prevent highestL2BlockNumber from being negative.
+        // highestL2BlockNumber isn't used anywhere, so it's safe to set this to 0.
```

Comment:
By "isn't used anywhere" I mean none of our code actually relies on this value. We could probably delete it, honestly. But whatever.